plugins/inputs/smart/smart.go

```diff
@@ -424,6 +405,29 @@ func parseInt(str string) int64 {
 	return 0
 }
 
+func parseCommaSeperatedInt(fields, deviceFields map[string]interface{}, str string) error {
```

Comment:
replace `deviceFields` name with an `_`, since it isn't being used in the function?